docs/apache-airflow/templates-ref.rst

```diff
@@ -81,6 +81,18 @@ Variable                                    Type                  Description
     The DAG run's logical date, and values derived from it, such as ``ds`` and
     ``ts``, **should not** be considered unique in a DAG. Use ``run_id`` instead.
 
+Accessing Airflow context variables from TaskFlow tasks 
+-------------------------------------------------------
+
+While ``@task`` decorated tasks don't support rendering jinja templates passed as arguments, 
```

Comment:
Not sure it makes sense to lead off talking about templating being unsupported in a section about context vars.  Might be better to go straight into the content of the section.  

Maybe add a section or a `.. note:` mentioning that templating is not supported in taskflow.

Reply:
Because this page is mostly in the context of Jinja, I wanted to explicitly point out that it doesn't work this way with TaskFlow.